### PR TITLE
Fixes for queries

### PR DESCRIPTION
**Pull request by turbo124**
set draft balances to amount - fixes for queries when creating payments.

### CODE REVIEW

**Comment by beganovich** [comment]
I'd assume this is QA passed?

**Comment by turbo124** [comment]
thanks!

File: src/pages/invoices/common/hooks/useResolveTotalVariable.tsx [hunk 1]
```diff
@@ -116,7 +105,7 @@ export function useResolveTotalVariable(props: Props) {
     if (variable == '$balance_due' && invoiceSum) {
       return (
         <Element leftSide={resolveTranslation(variable, '$')}>
-          {formatMoney(invoiceSum.invoice.balance)}
+          {invoiceSum.invoice.status_id === '1' ? formatMoney(invoiceSum.invoice.amount) :formatMoney(invoiceSum.invoice.balance)}
```
Comment (beganovich) [inline src/pages/invoices/common/hooks/useResolveTotalVariable.tsx:108]: Needs fix: Use the status enum.

**Review by beganovich — CHANGES_REQUESTED** [review]
Looks good, a tiny change needed :+1:

**Pull request merged**